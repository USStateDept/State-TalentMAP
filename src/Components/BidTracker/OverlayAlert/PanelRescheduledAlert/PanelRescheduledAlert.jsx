import PropTypes from 'prop-types';
import FontAwesome from 'react-fontawesome';

const PanelRescheduledAlert = ({ date }) => (
  <div className="bid-tracker-alert-container bid-tracker-alert-container--in-panel">
    <div className="top-text">
      <div>
        <FontAwesome name="users" />
      </div>
      <div>
        {`You've been rescheduled for the next panel on ${date}`}
      </div>
      <div className="date-text">
        {date}
      </div>
    </div>
  </div>
);

PanelRescheduledAlert.propTypes = {
  date: PropTypes.string.isRequired,
};

export default PanelRescheduledAlert;
