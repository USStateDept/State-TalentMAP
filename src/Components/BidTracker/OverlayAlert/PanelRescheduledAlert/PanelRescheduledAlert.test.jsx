import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import PanelRescheduledAlert from './PanelRescheduledAlert';

describe('PanelRescheduledAlertComponent', () => {
  it('is defined', () => {
    const wrapper = shallow(
      <PanelRescheduledAlert date="6.11.17" />,
    );
    expect(wrapper).toBeDefined();
  });

  it('matches snapshot', () => {
    const wrapper = shallow(
      <PanelRescheduledAlert date="6.11.17" />,
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
