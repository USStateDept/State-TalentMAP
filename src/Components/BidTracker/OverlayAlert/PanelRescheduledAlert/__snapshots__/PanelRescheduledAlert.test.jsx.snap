// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PanelRescheduledAlertComponent matches snapshot 1`] = `
<div
  className="bid-tracker-alert-container bid-tracker-alert-container--in-panel"
>
  <div
    className="top-text"
  >
    <div>
      <FontAwesome
        name="users"
      />
    </div>
    <div>
      You've been rescheduled for the next panel on 6.11.17
    </div>
    <div
      className="date-text"
    >
      6.11.17
    </div>
  </div>
</div>
`;
